src/renderer/scss/component/_card.scss

```diff
@@ -1,367 +1,140 @@
 .card {
-  border-radius: var(--card-radius);
-  display: flex;
-  flex-direction: column;
-  margin-right: auto;
-  margin-left: auto;
-  max-width: var(--card-max-width);
-  user-select: text;
-  white-space: normal;
-
-  // .card-media__internal__links should always be inside a card
-  .card-media__internal-links {
-    top: $spacing-vertical * 2/3;
-    right: $spacing-vertical * 2/3;
-    position: absolute;
-  }
+  background-color: $lbry-white;
+  border: 1px solid $lbry-gray-1;
+  border-radius: 0.5rem;
+  margin-bottom: 1.5rem;
+  position: relative;
 
-  .card__media {
-    padding-top: var(--video-aspect-ratio);
+  html[data-theme='dark'] & {
+    background-color: rgba($lbry-white, 0.1);
+    border-color: rgba($lbry-white, 0.1);
   }
 
-  // Text that is shown if a piece of content has no thumbnail
-  // We need to do this because the thumbnail uses padding-top: var(--video-aspect-ratio); for dynamic height
-  // this lets the text sit in the middle instead of the bottom
-  .card__media-text {
-    margin-top: calc(var(--video-aspect-ratio) * -1);
+  &--disabled {
+    opacity: 0.3;
+    pointer-events: none;
   }
-}
-
-.card--disabled {
-  opacity: 0.3;
-  pointer-events: none;
-}
-
-.card--link {
-  cursor: pointer;
-}
-
-.card--pending {
-  opacity: 0.5;
-}
-
-.card--section {
-  margin-top: $spacing-vertical * 2/3;
-  padding: $spacing-vertical;
-  background-color: $lbry-white;
-}
 
-.card--small {
-  font-size: 13px;
-}
+  &--link {
+    cursor: pointer;
+  }
 
-.card--space-between {
-  display: flex;
-  justify-content: space-between;
-}
+  &--section {
+    padding: 2rem;
 
-.card--wallet-balance {
-  background: url('../../../static/img/stripe-background.png') no-repeat;
-  background-size: cover;
-  color: $lbry-white;
-  justify-content: space-between;
+    .card__content:not(:last-child) {
+      margin-bottom: 1.5rem;
```

Comment:
Why the change the `margin-bottom` instead of `margin-top`? IME always using `margin-top`/`margin-left` makes it a lot simpler to add new pieces of ui, because you don't rely on the markup before your new component to provide the spacing. 

It also requires removing the margin on `:last-child` everywhere.

Reply:
Copying from Bootstrap's docs:

> Avoid margin-top. Vertical margins can collapse, yielding unexpected results. More importantly though, **a single direction of margin is a simpler mental model**.

It's consistent with how the rest of the app is styled. Also, the removal of the margin on `:last-child` is only inside this block. I haven't seen anything to indicate visual issues.

Reply:
👍 I'm fine moving to `margin-bottom`